src/forms/Form/Form.stories.tsx

```diff
@@ -1122,3 +1128,185 @@ export const DefaultValues = (): ReactElement => {
 		</>
 	);
 };
+
+const FakeTopBar = function(props: { variant: string, openNav : MouseEventHandler }): ReactElement {

```

Comment:
All of this piece of code could be easily reduced if we just export them from LeftNav.stories.tsx, there's no need for this to be copied and pasted here if it's the same as that other file. The only thing that changes between them is the "<Lorem />" but that can easily be just a prop that gets passed down in the "DMS Example story". Let's try to follow the DRY principle as much as possible (Don't Repeat Yourself).